skorch/tests/test_utils.py

```diff
@@ -174,6 +178,25 @@ def test_tuple(self, to_numpy, x_tuple):
         for entry_numpy, entry_torch in zip(x_numpy, x_tuple):
             self.compare_array_to_tensor(entry_numpy, entry_torch)
 
+    def test_dict(self, to_numpy, x_dict):
+        x_numpy = to_numpy(x_dict)
+        self.compare_array_to_tensor(x_numpy['a'], x_dict['a'])
+        self.compare_array_to_tensor(x_numpy['b'][0], x_dict['b'][0])
+        self.compare_array_to_tensor(x_numpy['b'][1], x_dict['b'][1])
+
+    @pytest.mark.parametrize('x_invalid', [
+        1,
+        [1,2,3],
+        (1,2,3),
+        {'a': 1},
+    ])
+    def test_invalid_inputs(self, to_numpy, x_invalid):
+        " Inputs that are invalid for the scope of to_numpy. "
```

Comment:
I've never seen this style, neither docstring nor comment. I think a comment would be sufficient here?

Reply:
It is still a doc-string but according to PEP257 this style is disregarded. I converted it to a comment.

Example:
```python
>>> def foo():
...     "bla"
...     return 1
... 
>>> foo.__doc__
'bla'
```

Reply:
> It is still a doc-string but according to PEP257 this style is disregarded

Interesting, I didn't know, though in hindsight it makes sense.